docs/reference/query-dsl/geo-bounding-box-query.asciidoc

```diff
@@ -188,10 +188,10 @@ values separately.
         "filter" : {
             "geo_bounding_box" : {
                 "pin.location" : {
-                    "top" : -74.1,
-                    "left" : 40.73,
-                    "bottom" : -71.12,
-                    "right" : 40.01
+                    "top" : 40.73,
```

Comment:
With strict validation this does not work anymore as this means the bottom corner would be above the top corner.

Even with lenient validation I think the documentation is confusing to users as in all other examples of the same page latitude and longitude values are exactly the other way round.